### ISSUE TITLE
Wont send Jellyfin.wpt to tv

### ISSUE THREAD

**Issue by JMJEContent** [jeppevinkel/jellyfin-tizen-builds#26]
I simply want Jellyfin.app on Samsung TV QA43LS03AAWXXY. 
I run macOS and Tizen Studio just cannot locate JDK 
So I created VM Ubuntu 22.04 system and got Tizen Studio to operate and get to device manager:

<img width="689" alt="Screen Shot 2023-12-26 at 12 21 03 pm" src="https://github.com/jeppevinkel/jellyfin-tizen-builds/assets/154850233/de15e310-0743-43b6-aaec-b24074eaaa7c">

I've built a Samsung Certificate:

<img width="412" alt="Screen Shot 2023-12-26 at 12 22 39 pm" src="https://github.com/jeppevinkel/jellyfin-tizen-builds/assets/154850233/1798c5ba-d06f-4227-8b83-e0bbe61dbe2b">

I've setup Develop Mode on Samsung TV and can connect to the TV, have the Samsung cert approve

<img width="591" alt="Screen Shot 2023-12-26 at 12 24 45 pm" src="https://github.com/jeppevinkel/jellyfin-tizen-builds/assets/154850233/6f1cdc16-e0ef-41ed-ab0e-8e411b68fbdb">

select install App, select Jellyfin.wgt(latest version). It has the loading circle 

<img width="608" alt="Screen Shot 2023-12-26 at 12 26 31 pm" src="https://github.com/jeppevinkel/jellyfin-tizen-builds/assets/154850233/5b8bb0d6-01ec-4b5d-9b66-3a8d8e343874">

then nothing actually happens... no confirmation no Jellyfin app on tv. 

I then tried to manually install with the : tizen install -n Jellyfin.wgt -t QA43LS03aawxxy
error : tizen command not found

Any advice from here. thanks in advance

**Comment by JMJEContent** [comment]
SOLVED! by myself....
app is uploaded

in Ubuntu I had to Download Powershell,

And the code needed a week

./tizen.sh install -n <full file name of Jellyfin.wgt> -t <Tv model>

**Comment by kq76** [comment]
I had a similar problem and was finally able to resolve it without having to resort to trying Linux and I figured I'd share in case someone else encounters the same problem. This is the first time I've tried doing anything with git (other than just downloading stuff). So if someone is like me, don't give up. If I figured it out, you probably can too.

I first tried Jeppe's instructions, but got stuck somewhere so I tried the long instructions (https://github.com/jellyfin/jellyfin-tizen). But then I got stuck in the "Build Jellyfin Web" part and couldn't figure my way out of it. I do wonder if Jeppe's instructions assume you did the first half of those longer instructions (the Tizen stuff). 

So I went back to Jeppe's instructions and, thanks to the OP here, I finally got it by changing the PowerShell instruction to ".\tizen install -n Jellyfin.wgt -t UN50KU6270" (that last part will most likely be different for others). You also need to run PowerShell from the Tizen folder and have the .wgt file in it. I thought maybe I'd have to enter that in git, but you don't, PowerShell works.

Thanks Jeppe!
